vault/programs/vault/src/context.rs

```diff
@@ -0,0 +1,323 @@
+use crate::zeta_context::*;
+use crate::*;
+
+#[derive(Accounts)]
+#[instruction(vault_name: String, vault_lamports: u64, bumps: VaultBumps)]
+pub struct InitializeVault<'info> {
+    // vault Authority accounts
+    #[account(mut)]
+    pub vault_authority: Signer<'info>,
+    // vault Accounts
+    #[account(init,
+        seeds = [vault_name.as_bytes()],
+        bump = bumps.vault,
+        payer = vault_authority)]
+    pub vault: Box<Account<'info, Vault>>,
+    // This is the PDA that holds SOL to pay for the margin account
```

Comment:
vault_wallet probably a better name

Reply:
Changing it such that:
`vault_authority` -> `admin`
`vault_payer` -> `vault_authority`
in line with your naming conventions from flex